### DISCUSSION TITLE
Add new device not possible and error "subscription ...  has expired"

### DISCUSSION THREAD

**Discussion by jove01** [make-all/tuya-local#1679] [Q&A]
Hi 

I want to add a new device to local tuya integration. 
![image](https://github.com/make-all/tuya-local/assets/116289742/dab0394e-4035-4622-b29f-159af318b061)

But there is no local key
![image](https://github.com/make-all/tuya-local/assets/116289742/a46d25b3-a07f-42be-929f-5a043d09e1a9)

so I looked into the IOT development plattform in my project to see the local key there. My device I found
![image](https://github.com/make-all/tuya-local/assets/116289742/f220fe2b-12ba-4dc7-9d34-585e158b501a)

But in the device detail there is an error
`{
  "code": 28841002,
  "msg": "No permissions. Your subscription to cloud development plan has expired.",
  "success": false,
  "t": 1708637869996,
  "tid": "a063cb30d1ca11eea1957edaaa357452"
}`

So I did not know, is the doorbell supported and an issue there, or is the permission the reason or have I done an error?

I didn't know the exact device information, because the doorbell is integrated in my letterbox.

I hope, you can help

**Comment by make-all** [comment]
This is a different integration, we don't support discovery, so that first screenshot does not exist in this integration.
